src/store/modules/user/types.ts

```diff
@@ -0,0 +1,23 @@
+
+/*----------------------- State -----------------------*/
+/** User State*/
+export interface UserState {
+  currentUser: string;
+  currentFarmer: string;
```

Comment:
I'm not sure I fully understand what this state is supposed to be modelling? If it's supposed to model the currently user that is logged in, this should probably look similar to the following should it not?
```
interface UserState {
  uuid: string;
  username: string;
  // Other properties we know about a trader
}
```

If we are viewing a current farmer and need to know that information in the store (I'm not totally sure we would need that considering the farmers are already in the store), it would belong in a different module. If you intend this to be for the farmers tabs so we know which farmer we are on, it should probably be in a module like `farmerTabs` or something similar?

Reply:
I intended this to hold the uuid of the farmer that has been selected from the list as well as the trader uuid so we have it for the collection/loans/buy and Info pages. would there be a better way to pass the farmer and trader UUID around?

Reply:
Ahh I understand better what you are trying to achieve. Hmm, maybe we can keep this all in the same module state but I think it would need to be renamed. Maybe `ActivePeople` or something?